### DISCUSSION TITLE
PyScript and rich ecosystem of pluggable components

### DISCUSSION THREAD

**Discussion by pauleveritt** [Finistere/antidote#50] [Ideas]
During PyCon, Flavio had an open space about PyScript. He has a vision of a rich ecosystem of pluggable components. I know it sounds exactly what I say about Sphinx and Python modern web. But the words came out of his mouth!

He also demo'd the start of his vision for PyScript's execution model. He wants reactive programming and injection of state, to avoid a big pile of scope. He's a big fan of the traits library which has an [observable concept](https://docs.enthought.com/traits/traits_tutorial/introduction.html#the-observe-decorator)

**Comment by Finistere** [comment]
I'm taking a look at the library, thanks a lot! :)
I agree that managing some states instead of scopes is better, and that is what I'd like for Antidote. I have yet to formalize it...
